### PR TITLE
Add PowerShell

### PR DESCRIPTION
**Pull request by chantisnake**
Previous PR: https://github.com/sindresorhus/awesome/pull/764

**https://github.com/janikvonrotz/awesome-powershell**

**A curated list of delightful PowerShell modules and resources **

# ALL THE BELOW CHECKBOXES SHOULD BE DONE AND CHECKED AND THE ABOVE PLACEHOLDERS SHOULD BE FILLED IN BEFORE SUBMITTING A PULL REQUEST!

- [x] I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- [x] This pull request has a descriptive title. *(For example: `Add Node.js`)*
- The list I added
	- [x] **has been around for at least 20 days,**
	- [x] is a non-generated Markdown file in a GitHub repo,
	- [x] is not a duplicate,
	- [x] only has awesome items with descriptions *(Awesome lists are curations of the best, not of everything)*,
	- [x] contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- [ ] has a Table of Contents section named `Contents` as the first section,
	- [x] has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)*,
	- [x] has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- [ ] has been checked for inconsistent formatting, spelling, and grammar *(This includes starting each link description with an uppercase character and ending it with a period `.`, and consistent naming)*,
	- [x] has been added at the bottom of the appropriate category.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -303,6 +303,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [SSH](https://github.com/moul/awesome-ssh)
 - [FOSS for Developers](https://github.com/httpsGithubParty/FOSS-for-Dev)
 - [HyperTerm](https://github.com/bnb/awesome-hyperterm)
+- [PowerShell](https://github.com/janikvonrotz/awesome-powershell)
```
Comment (sindresorhus) [inline readme.md]: Add a succinct description. See other items with description for how to format it.

**Comment by sindresorhus** [comment]
Feedback:

- I've unchecked some checkboxes that are clearly not done. E.g. some descriptions are missing `.` at the end.
- Drop the `Inspired by the awesome list thing.` text.
- `contribute.md` => `contributing.md`
- Drop all the `A` / `An` prefixes in the descriptions.
- There should be a succinct summary at the top about the project.
- In `A curated list of delightful PowerShell packages and resources.`, `PowerShell` should be linkified to https://en.wikipedia.org/wiki/PowerShell
- Use the [SVG version](http://mirrors.creativecommons.org/presskit/buttons/88x31/svg/cc-zero.svg) of the CC0 license badge.
- `Misc` section should be at the bottom.
- Put the `SharePoint` and `SQL Server` sections at the bottom (above `Misc`).

**Comment by sindresorhus** [comment]
ping

-

Merge conflict also needs to be fixed.

**Comment by chantisnake** [comment]
yes, I submitted all your feed back to the original repo. I don't think there is any news.

I am filled up with course work and my own stuffs. I will close this and reopen a PR later.

**Pull request closed without merging**